packages/wallet/app/components/modals/SignerModal/SignMessageSigner.tsx

```
@@ -17,9 +19,41 @@ type TExternalProps = {
   reject: () => void;
 };
 
-const SignMessageSigner: React.FunctionComponent<TExternalProps> = ({ data: _data, ...rest }) => (
+const Titles: { [key in EJwtPermissions]: React.ReactNode } = {
+  [EJwtPermissions.CHANGE_EMAIL_PERMISSION]: (
+    <FormattedMessage id="wallet.message-signer.title.change-email" />
+  ),
+  [EJwtPermissions.SUBMIT_KYC_PERMISSION]: (
+    <FormattedMessage id="wallet.message-signer.title.submit-kyc" />
+  ),
+  [EJwtPermissions.SUBMIT_ETO_PERMISSION]: (
+    <FormattedMessage id="wallet.message-signer.title.submit-eto" />
+  ),
+  [EJwtPermissions.UPLOAD_IMMUTABLE_DOCUMENT]: (
+    <FormattedMessage id="wallet.message-signer.title.upload-immutable-document" />
+  ),
+  [EJwtPermissions.DO_BOOK_BUILDING]: (
+    <FormattedMessage id="wallet.message-signer.title.do-book-building" />
+  ),
+  [EJwtPermissions.ISSUER_UPDATE_NOMINEE_REQUEST]: (
+    <FormattedMessage id="wallet.message-signer.issuer-update-nominee-request" />
+  ),
+  [EJwtPermissions.ISSUER_REMOVE_NOMINEE]: (
+    <FormattedMessage id="wallet.message-signer.issuer-remove-nominee" />
+  ),
+  [EJwtPermissions.SIGN_TOS]: null,
+  [EJwtPermissions.MANAGE_USER_WALLET]: null,
```

perhaps I should throw an error for these two?

---

do you know what's `MANAGE_USER_WALLET` is used for? I think SIGN_TOS we should support in case we disable the sign_tos permission that's active from the beginning but not sure about `MANAGE_USER_WALLET` as i don't see it's used in the code

---

Removed `MANAGE_USER_WALLET`